### PR TITLE
[IMP] pos: invoice issuance via qr code on receipts

### PR DESCRIPTION
**Pull request by LoredanaLrpz**
task ID: 3100959

### CODE REVIEW

File: content/applications/sales/point_of_sale/shop/invoice.rst [hunk 1]
```diff
@@ -58,6 +58,23 @@ To retrieve invoices from the **POS dashboard**,
...
    - **Invoiced orders** can be identified by the :guilabel:`Invoiced` status in the
-     :guilabel:`Status` column;
+     :guilabel:`Status` column.
    - You can filter the list of orders to invoiced orders by clicking :guilabel:`Filters` and
      :guilabel:`Invoiced`.
+
+QR codes to generate invoices
+-----------------------------
+
+Customers can also request an invoice by scanning the **QR code** displayed on their receipt. Upon
```
Comment (toaa-odoo) [inline content/applications/sales/point_of_sale/shop/invoice.rst:68]:
```suggestion
Customers can also request an invoice by scanning the **QR code** printed on their receipt. Upon
```

File: content/applications/sales/point_of_sale/shop/invoice.rst [hunk 2]
```diff
@@ -58,6 +58,23 @@ To retrieve invoices from the **POS dashboard**,
...
+scanning, they must fill in a form with their billing information and click :guilabel:`Get my
+invoice`. By doing so, the invoice is generated and available to be downloaded. In Odoo backend, the
```
Comment (toaa-odoo) [inline content/applications/sales/point_of_sale/shop/invoice.rst:70]:
```suggestion
invoice`. By doing so, the invoice is generated and available to be downloaded. In the backend of the POS app, the
```
Is there any a need to mention Odoo ? Do they ever leave Odoo for this feature ?

**Review by toaa-odoo — APPROVED** [review]
Approving already, but I made a few suggestions to consider :)

File: content/applications/sales/point_of_sale/shop/invoice.rst (hunk 2 above)
[the remark this answers is not in the document]
Comment (LoredanaLrpz) [reply content/applications/sales/point_of_sale/shop/invoice.rst:70]: I like your suggestion better

File: content/applications/sales/point_of_sale/shop/invoice.rst [hunk 3]
```diff
@@ -58,6 +58,24 @@ To retrieve invoices from the **POS dashboard**,
...
      :guilabel:`Invoiced`.
+
+QR codes to generate invoices
+-----------------------------
+
+Customers can also request an invoice by scanning the **QR code** printed on their receipt. Upon
+scanning, they must fill in a form with their billing information and click :guilabel:`Get my
+invoice`. On the one hand, doing so generates an invoice available for download. On the other hand,
+the order status goes from :guilabel:`Paid` or :guilabel:`Posted` to :guilabel:`Invoiced` in Odoo
+backend.
```
Comment (xpl-odoo) [inline content/applications/sales/point_of_sale/shop/invoice.rst:72]:
```suggestion
the order status goes from :guilabel:`Paid` or :guilabel:`Posted` to :guilabel:`Invoiced` in the Odoo
backend.
```

File: content/applications/sales/point_of_sale/shop/invoice.rst [hunk 4]
```diff
@@ -58,6 +58,24 @@ To retrieve invoices from the **POS dashboard**,
...
+
+.. image:: invoice/order-status.png
+   :align: center
+   :alt: order status change
+
+To allow this feature, you have to enable QR codes on receipts by going to :menuselection:`Point of
```
Comment (xpl-odoo) [inline content/applications/sales/point_of_sale/shop/invoice.rst:78]:
```suggestion
To enable this feature, you have to enable QR codes on receipts by going to :menuselection:`Point of
```
Allowing a feature doesn't sound right. But I could be wrong :)

**Review by xpl-odoo — APPROVED** [review]
Sorry for the delay, I assumed I didn't have to review these :)

**Review by AntoineVDV — APPROVED** [review]
@robodoo delegate+

**Comment by LoredanaLrpz** [comment]
@robodoo r+

**Pull request closed without merging**